guava/src/com/google/common/base/Functions.java

```diff
@@ -148,16 +146,15 @@ public String toString() {
     }
 
     @Override
-    public @org.checkerframework.checker.nullness.qual.Nullable V apply(@NullableDecl K key) {
-      @org.checkerframework.checker.nullness.qual.Nullable
```

Comment:
Why did you removed two annotations here?
 * from the return value
 * From the local variable `result`

Reply:
These were removed during auto-merge and I overlooked these changes. I'll fix this and other changes which I might find. Thanks!